README.md

```diff
@@ -99,18 +108,22 @@ make format check
 
 `make check` will run `make test` as well. To explicitly run tests, use `make test`
 
-## Development
+## Compiling
 
-To compile skywire directly from this repository for local testing and development
+To compile skywire directly from cloned git sources
 
 ```
 git clone https://github.com/skycoin/skywire
 cd skywire
 #for the latest commits, check out the develop branch
 git checkout develop
-make build
+make build1
```

Comment:
Typo I would assume.

Reply:
I hope we did not introduce multiple build targets like `build1` `build2`...

Reply:
> I hope we did not introduce multiple build targets like `build1` `build2`...

synth objected to the binaries populating in the repository root, he wanted these in `./build`

so `make build1` preserves the previous output / behavior of `make build`

`make build` now creates the binaries as specified inside `./build`